code/Controllers/CMSMain.php

```diff
@@ -838,7 +838,7 @@ public function SearchForm()
         $dateGroup->setTitle(_t('SilverStripe\\CMS\\Search\\SearchForm.PAGEFILTERDATEHEADING', 'Last edited'));
 
         // view mode
-        $viewMode = HiddenField::create('view', false, $this->ViewState());
+        $viewMode = HiddenField::create('view', false, 'listview');
```

Comment:
Instead of hard-coding this, we should make ViewState() default to listview, shouldn't we?